app/models/mixins/sysprep_template_mixin.rb

```diff
@@ -0,0 +1,18 @@
+module SysprepTemplateMixin
+  def allowed_sysprep_customization_templates(_options = {})
+    result = []
+    return result if (source = load_ar_obj(get_source_vm)).blank?
+
```

Comment:
@djberg96  You can check the `platform` of the source VM here and return an empty array if it is not `windows`.

Example call to the `platform` method:
https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_provision/options_helper.rb#L59

Reply:
@gmcculloug Ok, done. I tested it out, sure enough, the sysprep templates aren't visible for Linux images.